src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -21,44 +22,32 @@
 import org.team1540.base.power.PowerManager;
 import org.team1540.base.util.SimpleCommand;
 import org.team1540.robot2018.commands.TankDrive;
-import org.team1540.robot2018.commands.arms.JoystickArms;
 import org.team1540.robot2018.commands.auto.AutonomousProfiling;
 import org.team1540.robot2018.commands.auto.AutonomousProfiling.TrajectorySegment;
-import org.team1540.robot2018.commands.auto.AutonomousProfilingFast;
-import org.team1540.robot2018.commands.auto.AutonomousProfilingFast.TrajectorySegmentFast;
-import org.team1540.robot2018.commands.auto.DriveBackward;
-import org.team1540.robot2018.commands.auto.TurnLeftBackwardScale;
-import org.team1540.robot2018.commands.auto.TurnLeftBackwardSwitch;
-import org.team1540.robot2018.commands.elevator.JoystickElevator;
-import org.team1540.robot2018.commands.elevator.MoveElevatorToPosition;
-import org.team1540.robot2018.commands.elevator.MoveElevatorToPositionNoCurrent;
-import org.team1540.robot2018.commands.groups.FrontScale;
+import org.team1540.robot2018.commands.auto.DriveTimed;
+import org.team1540.robot2018.commands.elevator.MoveElevatorSafe;
 import org.team1540.robot2018.commands.groups.GroundPosition;
-import org.team1540.robot2018.commands.groups.HoldElevatorWrist;
-import org.team1540.robot2018.commands.groups.IntakeSequence;
-import org.team1540.robot2018.commands.intake.EjectAuto;
-import org.team1540.robot2018.commands.intake.EjectAutoSlow;
-import org.team1540.robot2018.commands.intake.EjectCube;
+import org.team1540.robot2018.commands.intake.Eject;
 import org.team1540.robot2018.commands.wrist.CalibrateWrist;
-import org.team1540.robot2018.commands.wrist.JoystickWrist;
-import org.team1540.robot2018.commands.wrist.MoveWristToPosition;
+import org.team1540.robot2018.commands.wrist.MoveWrist;
+import org.team1540.robot2018.subsystems.Arms;
 import org.team1540.robot2018.subsystems.ClimberWinch;
 import org.team1540.robot2018.subsystems.DriveTrain;
 import org.team1540.robot2018.subsystems.Elevator;
 import org.team1540.robot2018.subsystems.Intake;
-import org.team1540.robot2018.subsystems.IntakeArms;
 import org.team1540.robot2018.subsystems.Wrist;
 
 public class Robot extends IterativeRobot {
   public static final DriveTrain drivetrain = new DriveTrain();
   public static final Intake intake = new Intake();
-  public static final IntakeArms intakeArms = new IntakeArms();
+  public static final Arms arms = new Arms();
   public static final Elevator elevator = new Elevator();
   public static final Wrist wrist = new Wrist();
   public static final ClimberWinch winch = new ClimberWinch();
 
-  private Command emergencyDriveCommand = new TankDrive();
+  public static OI oi;
```

Comment:
As I mentioned replacing the constructor with a static initializer means that this wouldn't be necessary